FinSearchUnified/Tests/Subscriber/SubscriberTestCase.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace FinSearchUnified\Tests\Subscriber;
+
+use Enlight_Controller_Request_RequestHttp;
+use Enlight_Controller_Response_ResponseHttp;
+use FinSearchUnified\Tests\TestCase;
+use ReflectionException;
+use ReflectionMethod;
+
+class SubscriberTestCase extends TestCase
+{
+    /**
+     * @param string $controller
+     * @param Enlight_Controller_Request_RequestHttp $request
+     * @param Enlight_Controller_Response_ResponseHttp|null $response
+     *
+     * @return mixed
```

Comment:
Use `Enlight_Controller_Action` instead of `mixed` which is the base class for all controllers.